internal/worker/dbaccessor/worker.go

```diff
@@ -221,12 +224,20 @@ func NewWorker(cfg WorkerConfig) (*dbWorker, error) {
 			// that case we do want to cause the dbaccessor to go down. This
 			// will then bring up a new dqlite app.
 			IsFatal: func(err error) bool {
+				// If a database is dead we should not kill the worker.
+				if errors.Is(err, database.ErrDBDead) {
```

Comment:
So this says that if we kill the worker with `ErrDBDead`, the worker is not restarted and it is not fatal to use, the parent, right?